witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -29,17 +29,31 @@ import Dscp.Witness.Web.Types
 -- Logic
 ----------------------------------------------------------------------------
 
+-- | Check that source account has enough money to make transaction.
+ensureEnoughMoney :: WitnessWorkMode ctx m => TxWitnessed -> m ()
+ensureEnoughMoney tw = do
+    let tx     = twTx tw
+        spent  = txInValue tx
+        source = tiaAddr (txInAcc tx)
+    account <- fromMaybe def <$> getMempoolAccountMaybe source
+
+    when (aBalance account < coinToInteger spent) $
```

Comment:
Does this take fees into account?

Reply:
Right. I guess I'll wait until #125 get merged